gulp/tasks/concatJS.js

```diff
@@ -0,0 +1,18 @@
+var gulp = require('gulp');
+var concat = require('gulp-concat');
+var plumber = require('gulp-plumber');
+var handleErrors = require('../utils/handleErrors');
+var config = require('../config').concatJS;
+
+gulp.task('concatJS', ['cleanCss'], function() {
```

Comment:
This is a nit, but typing camelCase on the commandline makes my eye twitch 😉 . Can we stick to some sort of consistency using lowercase? A common convention for gulp is to use something like `gulp css:clean` or `glup css:lint` etc..